docs/content/development/endpoints/add-middleware.md

```diff
@@ -169,7 +144,7 @@ class CustomService extends TransactionBaseService {
 export default CustomService
 ```
 
-You can learn more about lifetime in the [Create Service documentation](../services/create-service.md#service-life-time).
+Notice that the `LIFE_TIME` property has been set to `Lifetime.SCOPED`. If you want to access new registrations in the dependency container within a service, you must set the lifetime of the service either to `Lifetime.SCOPED` or `Lifetime.TRANSIENT`.
```

Comment:
todo: only valid for core services since the custom services are transient by default